Dockerfile

```
@@ -1,7 +1,11 @@
 FROM eclipse-temurin:17 AS build
```

to simplify the copy this is needed:
```suggestion
#syntax=docker/dockerfile:1.7-labs

FROM eclipse-temurin:17 AS build
```